### ISSUE TITLE
django's order_with_respect_to is not supported

### ISSUE THREAD

**Issue by karyon** [vandersonmota/model_mommy#231]
models that use [order_with_respect_to](https://docs.djangoproject.com/en/1.8/ref/models/options/#order-with-respect-to) will cause the following error when making them with model_mommy:

```
Traceback (most recent call last):
  File "/vagrant/evap/student/tests.py", line 18, in test_user_cannot_vote_for_themselves
    question = mommy.make(Question, questionnaire=questionnaire)
  File "/usr/local/lib/python3.4/dist-packages/model_mommy/mommy.py", line 79, in make
    return mommy.make(**attrs)
  File "/usr/local/lib/python3.4/dist-packages/model_mommy/mommy.py", line 262, in make
    return self._make(commit=True, **attrs)
  File "/usr/local/lib/python3.4/dist-packages/model_mommy/mommy.py", line 312, in _make
    model_attrs[field.name] = self.generate_value(field)
  File "/usr/local/lib/python3.4/dist-packages/model_mommy/mommy.py", line 420, in generate_value
    raise TypeError('%s is not supported by mommy.' % field.__class__)
TypeError: <class 'django.db.models.fields.proxy.OrderWrt'> is not supported by mommy.
```

[here](https://github.com/fsr-itse/EvaP/blob/c1413ecced981845ccd0702d7b8b3d6eb8903256/evap/evaluation/models.py#L495) is a model that uses it.

i do not yet fully understand what this option is supposed to do and whether we actually need it, but i thought i should report this nevertheless :)

**Comment by vandersonmota** [comment]
thanks @karyon We'll take a look.

**Comment by karyon** [comment]
fyi, i have quickfixed this for me with the following code in settings.py:
```
MOMMY_CUSTOM_FIELDS_GEN = {
    'django.db.models.fields.proxy.OrderWrt': int,
}
```

**Comment by vandersonmota** [comment]
@karyon 
fixed: b0ed74c
